src/edgepi/adc/edgepi_adc.py

```diff
@@ -931,12 +930,13 @@ def set_config(
             `conversion_mode` (ConvMode): set conversion mode for ADC1.
             `override_updates_validation` (bool): set to True to skip update validation
         """
-        # pylint: disable=unused-argument
+        # pylint: disable=possibly-unused-argument
+        adc_1_ch  = self.__analog_in_to_adc_in_map[adc_1_analog_in]
+        adc_2_ch  = self.__analog_in_to_adc_in_map[adc_2_analog_in]
```

Comment:
use get, otherwise you will get key error when key is None:
```
        adc_1_ch  = self.__analog_in_to_adc_in_map.get(adc_1_analog_in)
        adc_2_ch  = self.__analog_in_to_adc_in_map.get(adc_2_analog_in)
```